client/src/components/Jokes.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import axios from 'axios';
+import styled from 'styled-components';
+import dad from '../img/dad.jpg';
+
+const JokeContainer = styled.div`
+  background-image: url(${dad});
+  background-size: cover;
+  color: white;
+  height: 100vh;
+  padding-top: 200px;
+  font-size: 3rem;
+
+  h2 {
+    text-decoration: underline;
+  }
+`
+
+class Jokes extends React.Component {
+  state = {
+    jokes: [],
+    currentJoke: null,
+    error: null,
+  };
+
+  componentDidMount() {
+    const ax = axios.create({
```

Comment:
This is really clean, I'm really surprised this isn't common knowledge when it comes to using axios. It seems so intuitive, great job reading the docs and finding a more efficient solution.